Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/SpeedLoaders/magnum.yml

```
@@ -157,26 +124,15 @@
   - type: BallisticAmmoProvider
     proto: CartridgeMagnumUranium
   - type: Sprite
-    sprite: Objects/Weapons/Guns/Ammunition/SpeedLoaders/Magnum/magnum_speed_loader.rsi
+    sprite: Objects/Weapons/Guns/Ammunition/SpeedLoaders/Pistol/pistol_speed_loader.rsi # Ronstation - modification
     layers:
       - state: base
         map: [ "enum.GunVisualLayers.Base" ]
-      - state: uranium-6
+      - state: base-6
         map: [ "enum.GunVisualLayers.Mag" ]
   - type: MagazineVisuals
-    magState: uranium
+    magState: base
```

Needs the #Ronstation - Modification comment.
```suggestion
    magState: base # Ronstation - modification
 ```